test/config/AppConfigSpec.scala

```diff
@@ -16,44 +16,102 @@
 
 package config
 
-import org.scalatest.matchers.must.Matchers
-import org.scalatest.flatspec.AnyFlatSpec
-import play.api.{Configuration, Environment}
-import uk.gov.hmrc.play.bootstrap.config.ServicesConfig
+import play.api.Application
+import utils.SpecBase
 
-class AppConfigSpec extends AnyFlatSpec with Matchers {
+class AppConfigSpec extends SpecBase {
 
-  class AppConfigScenario {
-    val env = Environment.simple()
-    val configuration = Configuration.load(env)
-    val servicesConfig = new ServicesConfig(configuration)
-  }
+  "url" should {
+
+    "remove left hand side slash" in new Setup {
+
+      import appConfig.URLSyntacticSugar
+
+      val url: String = "http://localhost/" / "abcd"
```

Comment:
both localhost and abcd can be moved into Setup if you're wanting to reduce string literals.